site/.env

```diff
@@ -5,3 +5,4 @@
 # ... env.test  - to override these env vars when running a TEST version
 # ... env.prod  - to override these env vars when running a PROD version
 NEXT_PUBLIC_ENABLE_LOGIN=false
+SITE_URL=https://www.saltdesignsystem.com/
```

Comment:
FYI, this is a cheeky fix for broken `sitemap.xml` that I've snuck into this PR. Not directly related to the foundations pages, but good to get out.